docs/docs/application-basics.md

```diff
@@ -21,19 +22,19 @@ You would use the following application definition (in JSON format) to describe
 }
 ```
 
-Note that `cmd` in the above example is the command that gets executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`.
+Note that `cmd` in the above example is the command that is executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`. <!-- this last sentence: why is it relevant to the user at this point? -->
 
 <p class="text-center">
   <img src="{{ site.baseurl }}/img/marathon-basic-0.png" width="800" height="612" alt="Marathon deployment example: simple bash command">
 </p>
 
-What happens here is that Marathon hands over execution to Mesos. Mesos executes each task in its own sandbox environment.
-The sandbox is a special directory on each slave that acts as the execution environment (from a storage perspective) and also contains relevant log files as well as `stderr` and `stdout` for the command being executed. See also the role of the sandbox in [debugging distributed apps](https://docs.mesosphere.com/tutorials/debugging-a-mesosphere-cluster/).
+What happens here <!-- where is here? I want to rephrase this to "When you launch an app, Marathon... --> is that Marathon hands over execution to Mesos. Mesos executes each task in its own sandbox environment.
+The sandbox is a special directory on each slave that acts as the execution environment (from a storage perspective <!-- what does this mean? -->) and contains relevant log files as well as `stderr` and `stdout` for the command being executed. See also the role of the sandbox in [debugging distributed apps](https://docs.mesosphere.com/tutorials/debugging-a-mesosphere-cluster/).
```

Comment:
Please rewrite. The sandbox is a directory that is created by Mesos for each task. The `stdout` and `stderr` stream is written to this location.